ingestors/kafka/src/main/scala/hydra/kafka/programs/KeyAndValueSchemaV2Validator.scala

```diff
@@ -74,7 +78,9 @@ class KeyAndValueSchemaV2Validator[F[_]: Sync] private (schemaRegistry: SchemaRe
         nullableKeyFieldsValidationResult ++
         defaultNullableValueFieldsValidationResult ++
         unsupportedLogicalTypesKey ++
-        unsupportedLogicalTypesValues
+        unsupportedLogicalTypesValues ++
+        validateRequiredKeyFieldsResult ++
+        validateRequiredValueFieldsResult
```

Comment:
these 2 required fields checks result can be represented as 1 check (look comment above)

Reply:
https://github.com/pluralsight/hydra/pull/803#discussion_r839388389